recipe/meta.yaml

```diff
@@ -1,41 +1,9 @@
-{% set version = "2.5.2" %}
+{% set version = "2.6.0" %}
 
 package:
   name: conda-env
   version: {{ version }}
 
-source:
-  fn: conda-env-{{ version }}.tar.gz
-  url: https://github.com/conda/conda-env/archive/v{{ version }}.tar.gz
-  sha256: 7ca59fa861a3145e94c0d1377bf0bd78735f19123234bd3fcf4024a8f8e8ce03
-
-build:
-  number: 0
-  script: python setup.py install
-  entry_points:
-    - conda-env = conda_env.cli.main:main
-
-requirements:
-  build:
-    - python
-
-  run:
-    - python
-
-test:
-  imports:
-    - conda_env
```

Comment:
So this test is obviously out. What should we do instead? Make sure it can't be imported? Something else?